libs/dh/wholesale/feature-settlement-reports/src/lib/drawer/dh-settlement-report-drawer.component.ts

```diff
@@ -73,21 +77,36 @@ import { DhSettlementReportsStatusComponent } from '../util/dh-settlement-report
         margin: var(--watt-space-ml);
         width: 60%;
       }
+
+      .card-grid-areas {
+        margin: var(--watt-space-ml);
+      }
     `,
   ],
   templateUrl: './dh-settlement-report-drawer.component.html',
 })
 export class DhSettlementReportDrawerComponent {
   drawer = viewChild.required<WattDrawerComponent>(WattDrawerComponent);
 
+  tableSource = new WattTableDataSource<DhSettlementReport['gridAreas'][0]>();
+
+  columns: WattTableColumnDef<string> = {
+    code: { accessor: (value) => value },
+  };
+
   report = input.required<DhSettlementReport | undefined>();
```

Comment:
Suggestion: Seems off that something required can also be undefined?

Reply:
`required` is related to the existence of the input when the component is used in a tamplate, not the types of value it can accept. Have removed the `required` to make it less confusing.